distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -321,6 +323,43 @@ public void initFacades() {
         catalogBundle = new CatalogBundle(serviceManager, adminConfig);
         securityPolicy = new SecurityPolicyConfigurator(serviceManager, configAdmin);
         urlResourceReaderConfigurator = new UrlResourceReaderConfigurator(configAdmin);
+        console = new KarafConsole(getServiceManager().getBundleContext(),
+                features,
+                sessionFactory);
+    }
+
+    public void waitForBaseSystemFeatures() {
+        try {
+            basePort = getBasePort();
+            getAdminConfig().setLogLevels();
+            getServiceManager().waitForRequiredApps(getDefaultRequiredApps());
+            getServiceManager().waitForAllBundles();
+            getCatalogBundle().waitForCatalogProvider();
+
+            getServiceManager().waitForHttpEndpoint(SERVICE_ROOT + "/catalog/query");
+            getServiceManager().waitForHttpEndpoint(SERVICE_ROOT + "/csw?_wadl");
+
+            //currently non of the test use this feature but
+            //it generates a lot of error/warnings so turning it off
+            getServiceManager().stopFeature(false, "catalog-core-backupplugin");
```

Comment:
Don't know that an itest i needed for this but if a test did need that feature it could turn it on itself.